pages/windows/findstr.md

```diff
@@ -0,0 +1,35 @@
+# findstr
+
+> Find specified text within one or more files.
+
+- Find space-separated string(s) in all files:
+
+`findstr "{{query}}" *`
+
+- Find space-separated string(s) in all files recursively:
```

Comment:
Let us use the `[]` idiom here. Since the option is `/s` for recursively, whereas `/r` means regular expressions, I think we should help users remember the options better.

So let's use `recur[s]ively`, `e[x]actly`